containers/container_implementation.md

```diff
@@ -29,31 +28,138 @@ Model containers must implement functionality consistent with this information.
     * 3: 64-bit Doubles
     * 4: Strings
     
+The following is an example construction of a *new container message* in Python:
+

```

Comment:
Format the python code blocks in triple backticks like the following to get Python syntax highlighting
````
```py
 # Code goes here
```
````